dlt/common/known_env.py

```diff
@@ -0,0 +1,25 @@
+"""Defines env variables that `dlt` uses independently of its configuration system"""
+
+DLT_PROJECT_DIR = "DLT_PROJECT_DIR"
```

Comment:
are these in the docs somewhere? I think people had been asking about how change the datadir etc.

Reply:
let's just link this
https://deploy-preview-1536--dlt-hub-docs.netlify.app/docs/api_reference/common/known_env

from our docs

Reply:
https://github.com/dlt-hub/dlt/issues/1546